shared_workflow/install_shared.py

```diff
@@ -108,15 +108,15 @@ def install_simulation(
     if stat_file_path is not None:
         sim_params_dict[SimParams.stat_file.value] = stat_file_path
 
-    nt = float(vm_params_dict["sim_duration"]) / root_params_dict["dt"]
-    if not isclose(nt, round(nt)):
-        logger.critical(
-            "Simulation dt does not match sim duration. This will result in errors during BB. Simulation duration must "
-            "be a multiple of dt. Ignoring fault. Simulation_duration: {}. dt: {}.".format(
-                vm_params_dict["sim_duration"], root_params_dict["dt"]
-            )
-        )
-        return None, None, None
+    # nt = float(vm_params_dict["sim_duration"]) / root_params_dict["dt"]
```

Comment:
UNCOMMENT THESE LINES JUST PUT SOME SAFETY AROUND IT. to: future me from: a very kind soul